example/autoprefixes.cpp

```diff
@@ -67,7 +67,7 @@ struct thing_base_unit : boost::units::base_unit<thing_base_unit, boost::units::
 struct euro_base_unit : boost::units::base_unit<euro_base_unit, boost::units::dimensionless_type, 5>
 {
   static constexpr const char* name() { return("EUR"); }
-  static constexpr const char* symbol() { return("�"); }
+  static constexpr const char* symbol() { return("€"); }
```

Comment:
The gihub UI shows these as the same, but if you look at https://patch-diff.githubusercontent.com/raw/boostorg/units/pull/28.diff you'll see they're encoded differently.